service/static/js/rest_api.js

```diff
@@ -48,6 +56,16 @@ $(function () {
         $("#customer_table").append(create_customer_results_header());
```

Comment:
I think we need to add something similar here for the address_table, otherwise I'm not sure that we risk that bdd tests could fail in some cases because past search results haven't been cleared out